<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>582</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="calc_tax_button">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>150</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Calculate Tax</string>
   </property>
  </widget>
  <widget class="QLabel" name="milabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>260</y>
     <width>291</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="price_box">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="tax_rate">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>110</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="results_window">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>190</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="excel_test_button">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>220</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>prueba excel</string>
   </property>
  </widget>
  <widget class="QTableView" name="mitableView">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>350</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
